shared/telemetry/instrumentation.py

```
@@ -0,0 +1,681 @@
+                            if isinstance(inner_stream, bytes):
+                                body = inner_stream
+                                logger.info(f"[OTEL DEBUG] _stream is bytes, len: {len(body)}")
+                            elif hasattr(inner_stream, "read"):
+                                # It's a file-like object (BytesIO), try to read and reset
+                                try:
+                                    current_pos = inner_stream.tell() if hasattr(inner_stream, "tell") else 0
+                                    body = inner_stream.read()
+                                    if hasattr(inner_stream, "seek"):
+                                        inner_stream.seek(current_pos)
+                                    logger.info(f"[OTEL DEBUG] Read from _stream: {type(body)}, len: {len(body) if body else 0}")
+                                except Exception as read_err:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)